src/ex4/server/middlewares/validation.js

```diff
@@ -0,0 +1,57 @@
+import { validationResult, checkSchema } from "express-validator";
```

Comment:
would extract all tasks related validation to another file as this middleware can be used once again if you'll have another entity you would wise to validate and has different schema